src/py42/clients/audit_logs.py

```diff
@@ -0,0 +1,41 @@
+class AuditLogsClient(object):
+    def __init__(self, audit_log_service):
+        self._audit_log_service = audit_log_service
+
+    def get_all(
+        self,
+        begin_time=None,
+        end_time=None,
+        event_types=None,
+        user_ids=None,
+        user_names=None,
+        user_ip_addresses=None,
+        affected_user_ids=None,
+        affected_user_names=None,
+    ):
+        """Retrieve audit logs, filtered based on given arguments.
+
+        Args:
+            begin_time (datetime, optional): A datetime.datetime instance for a given timestamp. Defaults to None.
```

Comment:
No other py42 API uses `datetime`. For consistency, these be unix timestamps.

Reply:
done.